srcpkgs/linux-wifi-hotspot/patches/makefile.patch

```diff
@@ -0,0 +1,24 @@
+--- a/src/Makefile
++++ b/src/Makefile
+@@ -1,7 +1,7 @@
+ CC=gcc
+ PKGCONFIG = $(shell which pkg-config)
+
+-CFLAGS=`pkg-config --cflags gtk+-3.0`
++CFLAGS_PKGCONF=`pkg-config --cflags gtk+-3.0`
+ LIBS=`pkg-config --libs gtk+-3.0 --libs x11` -lstdc++ -lpng -lqrencode
+
+ APP_NAME="wihotspot"
+@@ -32,10 +32,10 @@ resources.c: ui/glade/wifih.gresource.xml ui/glade/wifih.ui
+ 	@$(MAKE) -f $(THIS_FILE) wihotspot-gui
+
+ $(ODIR)/%.o: ui/%.c
+-	$(CC) -c -o $@ $< $(CFLAGS)
++	$(CC) -c -o $@ $< $(CFLAGS) $(CFLAGS_PKGCONF)
+
+ $(ODIR)/%.o: ui/%.cpp
+-	g++ -c -o $@ $<
++	$(CXX) -c -o $@ $< $(CFLAGS)
```

Comment:
My bad could you please change this `CFLAGS` to `CXXFLAGS`.

Reply:
After testing those changes It compiles just fine, but after creating an access point it failed to start .
I would suggest reverting back to previous flags .